tools/windows/install-help/cal/customactiondata.cpp

```diff
@@ -84,6 +84,13 @@ bool CustomActionData::parseUsernameData()
         computed_domain = computername;
         this->domainUser = false;
     } else {
+        WCHAR netBiosDomainName[256];
+        DWORD size = sizeof(netBiosDomainName)/sizeof(WCHAR);
+        if (DnsHostnameToComputerName(computed_domain.c_str(), netBiosDomainName, &size))
+        {
```

Comment:
Should probably at least log the error if DnsHostnameToComputerName() fails, for troubleshooting should that occur.